src/DDoSX/DomainClient.php

```diff
@@ -67,6 +68,32 @@ public function create(Domain $domain)
             });
     }
 
+    /**
+     * @param Domain $domain
+     * @return null
+     */
+    public function deploy(Domain $domain)
+    {
+        $this->post('v1/domains/' . $domain->name . '/deploy');
+
+        return null;
+    }
+
+    /**
+     * @param $domainName
+     * @return Ip
+     */
+    public function getIpByDomainName($domainName)
+    {
+        $response = $this->request("GET", 'v1/domains/' . $domainName . '/ip');
+        $body = $this->decodeJson($response->getBody()->getContents());
+
+        return new Ip([
```

Comment:
We could use the new  `apiToFriendly` method here

Reply:
updated